src/segment-loader.js

```diff
@@ -717,6 +717,43 @@ export default class SegmentLoader extends videojs.EventTarget {
     };
   }
 
+  abortRequestEarly_(stats) {
+    const firstByteReceived = Date.now() - stats.firstByteReceived;
+    // Since bandwidth values reported by progress events stabilize as time goes on,
+    // we adjust bandwidth by a conservative ammount, increasing as more time has passed
+    // maxing at 0.8 to prevent aborting a request we actually do have enough bandwidth
+    // for.
+    const playlistBandwidthAdjustment = Math.min(0.8, firstByteReceived / 5000);
```

Comment:
Maybe a hard number is OK. Since it is dependent on ramp up time for requests, that should be consistent regardless of segment size. Have we seen a max amount of time for requests to ramp up to full bandwidth?